### ISSUE TITLE
Extend to Multilabel

### ISSUE THREAD

**Issue by aditya1709** [henry8527/COT#4]
Have you done any research/ have any inputs if someone wanted to extend this to a Multilabel problem? 
It would be basically replacing CrossEntropy with BinaryCrossEntropy. For COT we could just use output of a class after sigmoid instead of its softmax probability. 
Does it make sense?

**Comment by qpy7777777** [comment]
The loss calculated by the COT loss function code is negative, do I need to add the negative sign in the formula?
